src/telegram.ts

```diff
@@ -19,14 +19,20 @@ class Telegram extends ApiClient {
   }
 
   /**
-   * Get download link to a file
+   * Get download link or path to a file
    */
   async getFileLink(fileId: string | tt.File) {
     if (typeof fileId === 'string') {
       fileId = await this.getFile(fileId)
     } else if (fileId.file_path === undefined) {
       fileId = await this.getFile(fileId.file_id)
     }
+
+    // Local bot API instances return the absolute path to the file
+    if (fileId.file_path?.startsWith('/')) {
+      return fileId.file_path
```

Comment:
Why not return a `file:` URL?
```suggestion
      return 'file://' + fileId.file_path
```